### ISSUE TITLE
MC 1.10.2: java.lang.StackOverflowError: Ticking entity

### ISSUE THREAD

**Issue by TomberWolf** [SpongePowered/SpongeForge#1615]
- SpongeForge version: spongeforge-1.10.2-2281-5.2.0-BETA-2422
- Forge version: forge-1.10.2-12.18.3.2297-universal
- Java version: Build 1.8.0_131-b11
- Operating System:
 - [ ] Linux (Ubuntu, Debian, Fedora, etc)
 - [ ] macOS
 - [x] Windows Server 2012 R2

https://pastebin.com/x2Dzp4N0

**Comment by ryantheleach** [comment]
Copying and pasting mod list to enhance search power of github issues:

```
 Disabled 'E' = Errored
    UCHIJAAAA   mcp{9.19} [Minecraft Coder Pack] (minecraft.jar)
    UCHIJAAAA   FML{8.0.99.99} [Forge Mod Loader] (forge-1.10.2-12.18.3.2297-universal.jar)
    UCHIJAAAA   Forge{12.18.3.2297} [Minecraft Forge] (forge-1.10.2-12.18.3.2297-universal.jar)
    UCHIJAAAA   NotEnoughItems{2.1.3.220} [Not Enough Items] (NotEnoughItems-1.10.2-2.1.3.220-universal.jar)
    UCHIJAAAA   spongeapi{5.2.0-SNAPSHOT-c675e80} [SpongeAPI] (minecraft.jar)
    UCHIJAAAA   sponge{1.10.2-2281-5.2.0-BETA-2422} [SpongeForge] (spongeforge-1.10.2-2281-5.2.0-BETA-2422.jar)
    UCHIJAAAA   ccl-entityhook{1.0} [ccl-entityhook] (CodeChickenLib-1.10.2-2.5.9.267-universal.jar)
    UCHIJAAAA   ItemBlacklist{1.2.0.11} [ItemBlacklist] (ItemBlacklist-1.9.4-1.2.0.11.jar)
    UCHIJAAAA   nucleus-mixin{0.25.2-5.1} [Nucleus Mixins] (NucleusMixins-0.25.2-5.1.jar)
    UCHIJAAAA   forgeendertech{1.10.2-1.3.3.0} [Forge Endertech] (AdChimneys-ForgeEndertech-1.10.2-1.3.3.0-build.0055.jar)
    UCHIJAAAA   adchimneys{1.10.2-1.2.5.0} [Advanced Chimneys] (AdChimneys-1.10.2-1.2.5.0-build.0050.jar)
    UCHIJAAAA   koresample{1.10.2-L1.2} [Kore Sample] (anchienttreesl-koresample-1.10.2-L1.2.jar)
    UCHIJAAAA   chisel{MC1.10.2-0.0.11.25} [Chisel] (Chisel-MC1.10.2-0.0.11.25.jar)
    UCHIJAAAA   Chameleon{1.10-2.2.2} [Chameleon] (StorageDrawers-Chameleon-1.10-2.2.2.jar)
    UCHIJAAAA   StorageDrawers{1.10.2-3.7.9} [Storage Drawers] (StorageDrawers-1.10.2-3.7.9.jar)
    UCHIJAAAA   dendrology{1.10.2-L1.2.1} [Ancient Trees] (anchienttreesl-dendrology-1.10.2-L1.2.1.jar)
    UCHIJAAAA   animania{1.2} [Animania] (animania-1.10.2-1.2.jar)
    UCHIJAAAA   antiqueatlas{4.4.5} [Antique Atlas] (antiqueatlas-1.9.4-4.4.5.jar)
    UCHIJAAAA   JEI{3.14.7.420} [Just Enough Items] (jei_1.10.2-3.14.7.420.jar)
    UCHIJAAAA   AppleCore{2.2.0} [AppleCore] (AppleCore-mc1.10.2-2.2.0.jar)
    UCHIJAAAA   appleskin{1.0.5} [AppleSkin] (AppleSkin-mc1.10.2-1.0.5.jar)
    UCHIJAAAA   arsmagica2{1.5.018} [Ars Magica 2] (ArsMagica2-1.10.2-1.5.0-18.jar)
    UCHIJAAAA   AnimationAPI{1.2.4} [AnimationAPI] (ArsMagica2-1.10.2-1.5.0-18.jar)
    UCHIJAAAA   AutoSapling{1.3.3} [AutoSapling] (Autosapling-1.3.3-MC1.10.jar)
    UCHIJAAAA   battletowers{1.5.8} [Battle Towers] (BattleTowers-1.10.2.jar)
    UCHIJAAAA   Baubles{1.3.10} [Baubles] (Baubles-1.10.2-1.3.10.jar)
    UCHIJAAAA   mantle{1.10.2-1.1.5.205} [Mantle] (TConstruct-Mantle-1.10.2-1.1.5.jar)
    UCHIJAAAA   natura{1.10.2-4.1.0.65} [Natura] (natura-1.10.2-4.1.0.65.jar)
    UCHIJAAAA   tconstruct{1.10.2-2.6.5.10} [Tinkers' Construct] (TConstruct-1.10.2-2.6.5.jar)
    UCHIJAAAA   betterwithmods{0.14.24 Beta} [Better With Mods] (Better With Mods-0.14.24-1.10.2.jar)
    UCHIJAAAA   BiomeTweaker{2.1.230} [BiomeTweaker] (BiomeTweaker-1.10.2-2.1.230.jar)
    UCHIJAAAA   BiomesOPlenty{5.0.0.2217} [Biomes O' Plenty] (BiomesOPlenty-1.10.2-5.0.0.2217-universal.jar)
    UCHIJAAAA   bloodmoon{1.5.1} [Bloodmoon] (Bloodmoon-MC1.10.2-1.5.1.jar)
    UCHIJAAAA   Botania{r1.9-341} [Botania] (Botania r1.9-341.jar)
    UCHIJAAAA   carpentersblocks{3.4.001 POC} [Carpenter's Blocks] (Carpenter's Blocks v3.4.001 POC - MC 1.10.2.jar)
    UCHIJAAAA   chiselsandbits{12.12} [Chisels & Bits] (chiselsandbits-12.12.jar)
    UCHIJAAAA   chunkgen{1.3} [ChunkGen] (chunkgen-1.10.2-1.5.0.jar)
    UCHIJAAAA   CodeChickenLib{2.5.9.267} [CodeChicken Lib] (CodeChickenLib-1.10.2-2.5.9.267-universal.jar)
    UCHIJAAAA   CodeChickenCore{2.4.1.102} [CodeChicken Core] (CodeChickenCore-1.10.2-2.4.1.102-universal.jar)
    UCHIJAAAA   crafttweakerjei{1.0.1} [CraftTweaker JEI Support] (CraftTweaker-1.10.2-3.0.25.jar)
    UCHIJAAAA   MineTweaker3{3.0.25} [MineTweaker 3] (CraftTweaker-1.10.2-3.0.25.jar)
    UCHIJAAAA   customachievements{1.10.2-0.2.5} [Custom Achievements] (customachievements-1.10.2-0.2.5.jar)
    UCHIJAAAA   customspawner{3.10.1} [DrZhark's CustomSpawner] (CustomMobSpawner 3.10.1.jar)
    UCHIJAAAA   customnpcs{1.10.2} [CustomNPCs] (CustomNPCs_1.10.2(08may17).jar)
    UCHIJAAAA   drpcore{0.2.0b} [Dark Roleplay Core] (Dark Roleplay Core Alpha-0.2.0b.jar)
    UCHIJAAAA   drpmedieval{0.1.8} [Dark Roleplay Medieval] (Dark Roleplay Medieval Alpha v0.1.9.1.jar)
    UCHIJAAAA   dldungeonsjdg{1.9.2} [Doomlike Dungeons] (DoomlikeDungeonsJBG-1.9.2-MC1.10.jar)
    UCHIJAAAA   bitdrawers{0.36} [Drawers & Bits] (DrawersBits-1.10.2-0.36.jar)
    UCHIJAAAA   mocreatures{10.0.6} [DrZhark's Mo'Creatures Mod] (DrZhark's Mo'Creatures Mod-10.0.6.jar)
    UCHIJAAAA   dsurround{3.4.5.6} [Dynamic Surroundings] (DynamicSurroundings-1.10.2-3.4.5.6.jar)
    UCHIJAAAA   bookshelf{1.4.4.347} [Bookshelf] (EnchantingPlus-Bookshelf-1.10.2-1.4.4.347.jar)
    UCHIJAAAA   eplus{4.1.0.117} [Enchanting Plus] (EnchantingPlus-1.10.2-4.1.0.117.jar)
    UCHIJAAAA   ExtraBitManipulation{1.10.2-2.3.3} [Extra Bit Manipulation] (ExtraBitManipulation-1.10.2-2.3.3.jar)
    UCHIJAAAA   fairylights{2.1.1} [Fairy Lights] (fairylights-2.1.1-1.10.2.jar)
    UCHIJAAAA   farseek{2.1} [Farseek] (Farseek-1.9.4-2.1.jar)
    UCHIJAAAA   com.boydti.fawe{3.5.1} [FastAsyncWorldEdit] (FastAsyncWorldEdit-forge110-17.04.19-92fef6f-646-12.3.11.jar)
    UCHIJAAAA   FloodLights{1.2.8-151} [Flood Lights] (FloodLights-1.9.4-1.2.8-151.jar)
    UCHIJAAAA   Waila{1.7.0} [Waila] (Waila-1.7.0-B3_1.9.4.jar)
    UCHIJAAAA   ftbl{0.0.0} [FTBLib] (FTBLib-1.1x-3.6.5.jar)
    UCHIJAAAA   ftbu{0.0.0} [FTBUtilities] (FTBUtilities-1.1x-3.6.5.jar)
    UCHIJAAAA   geographicraft{0.8.7} [GeographiCraft] (GeographiCraft-0.8.7.jar)
    UCHIJAAAA   GraveStone{1.0.6} [GraveStone] (GraveStone-1.10.2-Graves-1.0.6.jar)
    UCHIJAAAA   GraveStone-Extended{1.0.4} [GraveStone-Extended] (GraveStone-Extended-1.0.4.jar)
    UCHIJAAAA   hardcoredarkness{1.9} [Hardcore Darkness] (HardcoreDarkness-MC1.10.2-1.9.jar)
    UCHIJAAAA   hopperducts{1.4.8} [Hopper Ducts] (hopperducts-mc1.9.4-1.4.8.jar)
    UCHIJAAAA   llibrary{1.7.4} [LLibrary] (iceandfire-llibrary-1.7.4-1.10.2.jar)
    UCHIJAAAA   iceandfire{1.0.1} [Ice and Fire] (iceandfire-1.0.1.jar)
    UCHIJAAAA   inventorytweaks{1.62-dev-66-c082db3} [Inventory Tweaks] (InventoryTweaks-1.62-dev-66.jar)
    UCHIJAAAA   malisiscore{1.10.2-4.4.0} [MalisisCore] (malisiscore-1.10.2-4.4.0.jar)
    UCHIJAAAA   malisisdoors{1.10.2-5.2.0} [MalisisDoors] (malisisdoors-1.10.2-5.2.0.jar)
    UCHIJAAAA   RadixCore{1.10.2-2.1.3} [RadixCore] (RadixCore-1.10.2-2.1.3-universal.jar)
    UCHIJAAAA   MCA{1.10.2-5.2.3} [Minecraft Comes Alive] (MCA-1.10.2-5.2.3-universal.jar)
    UCHIJAAAA   minecolonies{@VERSION@} [MineColonies] (minecolonies-universal-1.10.2-0.8.2817.jar)
    UCHIJAAAA   mtrm{1.2.1.26} [MineTweakerRecipeMaker] (MineTweakerRecipeMaker-1.10.2-1.2.1.26.jar)
    UCHIJAAAA   moarsigns{4.2.2.8} [MoarSigns] (MoarSigns-1.10.2-4.2.2.8.jar)
    UCHIJAAAA   Morpheus{1.10.2-3.1.13} [Morpheus] (Morpheus-1.10.2-3.1.13.jar)
    UCHIJAAAA   harvestcraft{1.9.4-1.10.2g} [Pam's HarvestCraft] (Pam's HarvestCraft 1.9.4-1.10.2i.jar)
    UCHIJAAAA   limelib{1.5.3} [LimeLib] (particlespreader-limelib-1.10.2-1.5.3.jar)
    UCHIJAAAA   particlespreader{1.1.0} [Particle Spreader] (particlespreader-1.10.2-1.1.0.jar)
    UCHIJAAAA   plants{0.8.5} [Plants] (Plants-1.10.2-beta-0.8.5.jar)
    UCHIJAAAA   shetiphiancore{3.3.9} [ShetiPhian-Core] (platforms-shetiphiancore-1.10.0-3.3.9.jar)
    UCHIJAAAA   platforms{1.2.10} [Platforms] (platforms-1.10.0-1.2.10.jar)
    UCHIJAAAA   plustic{5.1.1.0} [PlusTiC] (plustic-5.1.1.0.jar)
    UCHIJAAAA   redstonepaste{1.7.4} [Redstone Paste] (redstonepaste-mc1.9.4-1.7.4.jar)
    UCHIJAAAA   roguelike{1.5.9} [Roguelike Dungeons] (RoguelikeDungeons-1.10.2-1.5.9.jar)
    UCHIJAAAA   ropebridge{2.0.5} [Rope Bridge] (ropebridge-1.10.2-2.0.5.jar)
    UCHIJAAAA   RTG{4.1.2.4} [Realistic Terrain Generation] (RTG-1.10.2-4.1.2.4.jar)
    UCHIJAAAA   ruins{16.2} [Ruins Mod] (Ruins-1.10.2.jar)
    UCHIJAAAA   rustic{0.1.2.2} [Rustic] (rustic-0.1.2.2.jar)
    UCHIJAAAA   shiftedsnow{${version}} [Better Snow] (shiftedsnow-0.2.jar)
    UCHIJAAAA   signpost{1.05.4} [Sign Post] (signpost-1.10.2-1.05.4.jar)
    UCHIJAAAA   SophisticatedWolves{3.6.1} [SophisticatedWolves] (SophisticatedWolves-3.6.5.jar)
    UCHIJAAAA   storagedrawersextra{1.10-1.1.0} [Storage Drawers Extras] (StorageDrawersExtras-1.10-1.1.0.jar)
    UCHIJAAAA   streams{0.3} [Streams] (Streams-1.9.4-0.3.jar)
    UCHIJAAAA   structpro{3.8} [StructPro] (structpro-3.8-1.10.2.jar)
    UCHIJAAAA   ToughAsNails{1.1.1.21} [Tough As Nails] (ToughAsNails-1.9.4-1.1.1.21-universal.jar)
    UCHIJAAAA   tanspit{@VERSION@} [Campfire Spit] (tanspit-1.10.2-1.3.jar)
    UCHIJAAAA   thebetweenlands{2.0.2-alpha} [The Betweenlands] (TheBetweenlands-2.0.2-alpha-universal.jar)
    UCHIJAAAA   thespotlightmod{1.1.0.88} [The SpotLight Mod] (TheSpotlightMod-1.1.0.88-legacy-mc1.10.2-universal.jar)
    UCHIJAAAA   thutcore{3.12.3} [ThutCore] (thutcore-3.12.3.jar)
    UCHIJAAAA   thuttech{@VERSION@} [Thut's Tech] (thuttech-4.0.10.c.jar)
    UCHIJAAAA   tombmanygraves{1.10.2-2.1.16} [Tomb Many Graves] (TombManyGraves-1.10.2-2.1.16.jar)
    UCHIJAAAA   undergroundbiomes{1.0.v.18} [Underground Biomes] (UndergroundBiomesConstructs-1.10-1.0.v.19.jar)
    UCHIJAAAA   warpscrolls{1.0} [warpscrolls] (Warp Scrolls 1.9.4 1.0.2.jar)
    UCHIJAAAA   waystones{2.0.12} [Waystones] (Waystones_1.10.2-2.0.12.jar)
    UCHIJAAAA   wearablebackpacks{1.7.0} [Wearable Backpacks] (WearableBackpacks-1.10.2-1.7.0.jar)
    UCHIJAAAA   conquest{0.0.1} [Conquest Reforged] (Wolfskraft Reforged-2.0.3-mc.1.10.2.jar)
    UCHIJAAAA   worldedit{6.1.4} [WorldEdit] (worldedit-forge-mc1.10.2-6.1.4-dist.jar)
    UCHIJAAAA   wtfcore{1.10.2_v1.5} [WTFs Expedition] (WTF-Expedition-WorldGen_1.10.2_-v1.5.1.jar)
    UCHIJAAAA   aliases{1.0} [Aliases] (Aliases-1.0.jar)
    UCHIJAAAA   bookotd{2.0.0} [BookOTD] (book-otd-2.0.0.jar)
    UCHIJAAAA   helpme{0.2.2} [HelpMe] (helpme-5.0.0-0.2.2.jar)
    UCHIJAAAA   mmcreboot{2.0.4} [MMCReboot] (MMCReboot-2.0.4-API-5.1-6.X.jar)
    UCHIJAAAA   mmcrules{1.1} [MMCRules] (MMCRules-1.1-API-5.0.0.jar)
    UCHIJAAAA   mmctickets{1.1.1} [MMCTickets] (MMCTickets-1.1.1-API-5.1-6.X.jar)
    UCHIJAAAA   nucleus{0.29.2-S5.1} [Nucleus] (Nucleus-0.29.2-S5.1-plugin.jar)
    UCHIJAAAA   permissionsex{2.0-SNAPSHOT-b147} [PermissionsEx] (PermissionsEx-Sponge-2.0.jar)
    UCHIJAAAA   pjp{0.13.6} [ProjectPortals] (projectportals-5.1.0-0.13.6.jar)
    UCHIJAAAA   redprotect{7.1.4} [RedProtect] (RedProtect-Sponge-6.0.0-7.1.4-b103.jar)
    UCHIJAAAA   ultimatecore{3.0.7} [UltimateCore] (UltimateCore-3.0.11.jar)
    UCHIJAAAA   statusprotocol{} [StatusProtocol] (UltimateCore-3.0.11.jar)
    UCHIJAAAA   hungeroverhaul{1.10.2-1.1.0.jenkins118} [Hunger Overhaul] (HungerOverhaul-1.10.2-1.1.0.jenkins118.jar)
    UD  BiomeTweakerCore{2.1.230} [BiomeTweaker Core] (minecraft.jar)
```

Seemingly caused by : EntityHenLeghorn

**Comment by gabizou** [comment]
Closing this issue, as of today 1.10.2 and 1.11.2 are not longer supported. If a sufficiently, decent, and tested Pull Request is made to fix this bug/issue, it will still be considered, however, it is NOT to be expected that any bug fixes made for 1.12.2 will be back ported to 1.11.2 or 1.10.2.

If you can reproduce this issue with SpongeForge on Minecraft 1.12.2 and this issue was wrongly closed, please [open a new issue](https://github.com/SpongePowered/SpongeForge/issues/new) with updated information.
